### ISSUE TITLE
Gracefully handle source regressions

### ISSUE THREAD

**Issue by pnoll1** [openaddresses/machine#772]
Machine accepts bad data from source making it the last successful run and what gets packaged in the data downloads.

Example:[Franklin County WA](https://results.openaddresses.io/sources/us/wa/franklin)  28,619 addresses to 37

Marking the run as failed and returning the previously cached result would be much better for data consumers.

**Comment by iandees** [comment]
Machine doesn't know that it's bad data, but it could check to see if the row count changed significantly and flag it as an error. I think the system that @ingalls is working on to replace machine should provide for this.

**Comment by pnoll1** [comment]
Is there a roadmap or public code to track progress on this?

**Comment by ingalls** [comment]
Yes, you can see the new processing system here: https://github.com/openaddresses/batch
